### PR TITLE
Fix build issue and unit-test exception issue on windows

### PR DESCRIPTION
**Pull request by ahnyoung-paul**
### Details:
 - *item1*
 - *...*

### Tickets:
 - *ticket-id*

### CODE REVIEW

File: inference-engine/thirdparty/clDNN/src/gpu/loop_gpu.cpp [hunk 1]
```diff
@@ -125,7 +125,7 @@ struct loop_gpu : typed_primitive_impl<loop> {
         auto& iteration_mem = instance.input_iteration_mem;
         auto body_network = instance.get_body_network();
         const auto& input_primitive_map = node.get_input_mappings();
-        const int inputs_memory_count = instance.inputs_memory_count();
+        const int inputs_memory_count = static_cast<int>(instance.inputs_memory_count());
```
Comment (yeonbok) [inline inference-engine/thirdparty/clDNN/src/gpu/loop_gpu.cpp]: I think we can set most stuffs with size_t

Comment (ahnyoung-paul) [reply inference-engine/thirdparty/clDNN/src/gpu/loop_gpu.cpp]: Okay, I just fix to build issue but i think so. I will change it.

File: inference-engine/thirdparty/clDNN/src/loop.cpp [hunk 2]
```diff
@@ -134,7 +134,7 @@ layout loop_inst::calc_output_layout(loop_node const & node) {
     if (axis_to_iterate_throgh != -1) {
         const auto shape = loop_output_layout.size.sizes(loop_output_layout.format);
         const size_t ndim = shape.size();
-        const size_t raw_axis = node.convert_to_raw_axis(axis_to_iterate_throgh, ndim);
+        const size_t raw_axis = node.convert_to_raw_axis(axis_to_iterate_throgh, (int)ndim);
```
Comment (yeonbok) [inline inference-engine/thirdparty/clDNN/src/loop.cpp]: this c-style casting is not acceptable. please use c++ style casting

Comment (ahnyoung-paul) [reply inference-engine/thirdparty/clDNN/src/loop.cpp]: Fixed it

File: inference-engine/thirdparty/clDNN/src/gpu/loop_gpu.cpp (hunk 1 above)
Comment (ahnyoung-paul) [reply inference-engine/thirdparty/clDNN/src/gpu/loop_gpu.cpp]: Fixed it

**Review by ghost — APPROVED** [review]

**Pull request merged**